src/app/(components)/Navbar.css

```
@@ -0,0 +1,16 @@
+.navbarContainer {
+    background-color: red;
+    display: flex;
+    justify-content: space-between;
+    align-items: center; /* Optional: align items vertically in the center */
+    flex-direction: row;
+    position: sticky;
+    top:0;
+    z-index: 30;
+  }
+  
+  .menuContainer {
+    display: flex;
+    flex-grow: 1;
+    justify-content: space-around;
+  }
```

Always end the file on an empty line

```suggestion
  }
  
```